themes/sweetpink.yaml

```diff
@@ -61,4 +62,4 @@ sweetpink:
   paper-slider-knob-start-color: "var(--purple)"
 
   switch-unchecked-color: "var(--teal)"
-  switch-unchecked-button-color: "var(--teal)"
\ No newline at end of file
+  switch-unchecked-button-color: "var(--teal)"
```

Comment:
sweet! we could refactor those switches to use the accent-color, but I am fine leaving them as individual colors, easier for people to see/modify individually that way